### PR TITLE
fix:login btn text alignment of navbar on small screen

### PR DESCRIPTION
**Pull request by Pavangbhat**
### Fixed text position of login button on small screen #137 

image

### CODE REVIEW

File: css/pages/navbar.css [hunk 1]
```diff
@@ -116,6 +119,7 @@ nav .menu {
   .menu {
     display: none;
   }
+
```
Comment (Pavangbhat) [inline css/pages/navbar.css:122]: Sorry for this it got placed by mistake

Comment (Rucha1499) [reply css/pages/navbar.css:122]: Saying about the extra line?

File: css/pages/navbar.css [hunk 2]
```diff
@@ -91,6 +91,9 @@ nav .menu {
     font-size: 25px;
     line-height: 0;
   }
+  .login-btn{
+    top: -10px;
```
Comment (swarajpure) [inline css/pages/navbar.css:95]: Doesn't adding `bottom: 10px` work? Just wondering 🤔

File: css/pages/navbar.css (hunk 1 above)
Comment (Pavangbhat) [reply css/pages/navbar.css:122]: yes

Comment (Rucha1499) [reply css/pages/navbar.css:122]: Prettier adds that! No worries. 😄

File: css/pages/navbar.css (hunk 2 above)
Comment (Pavangbhat) [reply css/pages/navbar.css:95]: not sure why it did not work @swarajpure 
image

Comment (swarajpure) [reply css/pages/navbar.css:95]: Can you please try margin/padding?

Comment (Pavangbhat) [reply css/pages/navbar.css:95]: image
@swarajpure  please have a look

**Review by swarajpure — APPROVED** [review]
Looks good to me!
Let @Rucha1499 have a look once.

**Review by Rucha1499 — APPROVED** [review]
Looks good to me! 
Thank you! 😃

**Pull request merged**